### PR TITLE
Pr vtol params

### PR DESCRIPTION
**Pull request by tumbili**
Fixes the bug which updated vtol parameters on every iteration. This should reduce CPU load drastically.
Furthermore, this should fix a memory leak.

### CODE REVIEW

**Comment by tumbili** [comment]
@julianoes Can you have a look please?

**Comment by LorenzMeier** [comment]
Jenkins test this please.

**Comment by dagar** [comment]
I can confirm that this decreases vtol_att_control cpu usage substantially. Now only ~ 4% on a pixhawk.

File: src/modules/vtol_att_control/vtol_att_control_main.h [hunk 1]
```diff
@@ -218,10 +218,10 @@ class VtolAttitudeControl
...
 	bool _abort_front_transition;
 
-	VtolType *_vtol_type;	// base class for different vtol types
-	Tiltrotor *_tiltrotor;	// tailsitter vtol type
-	Tailsitter *_tailsitter;	// tiltrotor vtol type
-	Standard *_standard;	// standard vtol type
+	VtolType *_vtol_type = nullptr;	// base class for different vtol types
+	Tiltrotor *_tiltrotor = nullptr;	// tailsitter vtol type
+	Tailsitter *_tailsitter = nullptr;	// tiltrotor vtol type
+	Standard *_standard = nullptr;	// standard vtol type
```
Comment (dagar) [inline src/modules/vtol_att_control/vtol_att_control_main.h]: This is a nitpick unrelated to your change, but is there a reason we need these? The single _vtol_type should be sufficient.

**Comment by tumbili** [comment]
@dagar Fixed!

**Comment by LorenzMeier** [comment]
Looks good to merge, do when it passes.

**Pull request merged**